tests/feature_elimination/test_feature_elimination.py

```diff
@@ -153,9 +203,9 @@ def test_get_feature_shap_values_per_fold(X, y):
 
 @pytest.mark.skipif(os.environ.get("SKIP_LIGHTGBM") == 'true', reason="LightGBM tests disabled")
 def test_complex_dataset(complex_data, complex_lightgbm):
+    
     X, y = complex_data
 
-
     param_grid = {
         'n_estimators': [5, 7, 10],
         'num_leaves': [3, 5, 7, 10],
```

Comment:
I would also add tests with Linear and non-linear models to the Shap resemblance model and Shap interpreter.

Reply:
Added tests to the interpret module and resemblance modules.